cookiecutter/vcs.py

```diff
@@ -15,34 +18,263 @@
 logger = logging.getLogger(__name__)
 
 
-BRANCH_ERRORS = [
-    'error: pathspec',
-    'unknown revision',
+class VCS:
+    """Abstract base VCS class. Handles the cloning of VCS repositories."""
+
+    cmd = 'xyz'
+    """VCS command"""
```

Comment:
Are comments after the attribute definition standard for this repository?

Reply:
It is at least standard Python and is used by editors and sphinx.